riscv/src/continuations/bootloader.rs

```diff
@@ -18,6 +18,9 @@ pub const MERKLE_TREE_DEPTH: usize = N_LEAVES_LOG + 1;
 pub const PAGE_SIZE_BYTES: usize = 1 << PAGE_SIZE_BYTES_LOG;
 pub const PAGE_NUMBER_MASK: usize = (1 << N_LEAVES_LOG) - 1;
 pub const BOOTLOADER_INPUTS_PER_PAGE: usize = WORDS_PER_PAGE + 1 + 4 + (MERKLE_TREE_DEPTH - 1) * 4;
+pub const MEMORY_HASH_START_INDEX: usize = REGISTER_NAMES.len();
+pub const NUM_PAGES_INDEX: usize = MEMORY_HASH_START_INDEX + 8;
+pub const PAGE_INPUTS_OFFSET: usize = NUM_PAGES_INDEX + 1;
```

Comment:
We should probably make this a trait at some point

Reply:
How exactly? You mean there is a trait with const generic parameters `MEMORY_SIZE_LOG`, `PAGE_SIZE_BYTES_LOG`, `BYTES_PER_WORD` and default implementations for all the derived values?

Reply:
just something that forces everything to be implemented in an organized way. But if we don't have more implementations that's not really needed I guess